shared/netutils/dhcpserver.h

```diff
@@ -31,6 +31,10 @@
 #define DHCPS_BASE_IP (16)
 #define DHCPS_MAX_IP (8)
 
+#ifndef DHCP_DEBUG
+#define DHCP_DEBUG    LWIP_DBG_ON
```

Comment:
won't this turn the debug messages back on? is that what you intended?

Reply:
When `DEBUG=0` the user shouldn't see them.
When `DEBUG=1` you probably want them.

Reply:
It builds (w & wo/ debug) but doesn't work though, and I am not sure why.